exporter.go

```diff
@@ -217,19 +215,18 @@ func (e *Exporter) saveImage(image imgutil.Image, additionalNames []string) erro
 		return idErr
 	}
 
-	e.logReference(id)
-	return saveErr
-}
-
-func (e *Exporter) logReference(identifier imgutil.Identifier) {
-	switch v := identifier.(type) {
-	case local.IDIdentifier:
-		e.Logger.Infof("\n*** Image ID: %s\n", v.String())
-	case remote.DigestIdentifier:
-		e.Logger.Infof("\n*** Digest: %s\n", v.Digest.DigestStr())
-	default:
-		e.Logger.Infof("\n*** Reference: %s\n", v.String())
+	refType, ref := getReference(id)
+	e.Logger.Info("*** Images:")
+	for _, n := range append([]string{image.Name()}, additionalNames...) {
+		if ok, message := getSaveStatus(saveErr, n); !ok {
+			e.Logger.Infof("      %s - %s\n", n, message)
+		} else {
+			e.Logger.Infof("      %s (%s)\n", n, TruncateSha(ref))
```

Comment:
It seems a little strange that we'll be displaying the same value over and over for each image tag. Not sure what a better solution would be. Just calling it out. :D

Reply:
I have mixed feelings and struggled with this. In some sense, it's good that each line stands on its own. And most people won't be building with more than one tag i imagine. I think like Ben's detect output, this is not an end state. We'll keep improving.